docker/appserver/Tomcat/example/Dockerfile

```diff
@@ -1,15 +1,13 @@
-FROM iaf-as-tomcat AS iaf-example-tomcat
+FROM iaf-webapp-tomcat AS iaf-example-tomcat
 
 # copy any additional drivers
 COPY target/dependencies/*.jar /usr/local/tomcat/lib/
 
-# copy war
-COPY target/dependencies/ibis-adapterframework-example.war /usr/local/tomcat/webapps/ROOT.war
+# copy resources
+COPY target/dependencies/resources /opt/frank/resources
 
-# copy configuration, properties and testscenarios
-#COPY target/dependencies/frank/configuration/*-resources.jar /opt/frank/resources/
-#COPY target/dependencies/frank/configuration/*-configurations.jar /opt/frank/configurations/
-#COPY target/dependencies/frank/testtool /opt/frank/testtool
+# copy context, normally this would be mounted from the environment, but for example a H2 database is used
```

Comment:
Ik snap dit commentaar niet. Bedoel je dat omdat de database intern is, de context dat ook wel kan zijn?